lblod/frontend-harvesting-self-service
# Feature/authentication

### description
Now we can harvest sensitive data on server because anyone can access harvesting-self-service and download that data. Solution is to add login as a feature flag because we want to use this app both with and without login screen

PR adds an authentication feature flag {{AUTHENICATION_ENABLED}} . Defaults to false, if true: Add basic login & allow authenticated jobs & scheduled jobs

To test:
- Setup frontend with following backend: https://github.com/lblod/app-lblod-harvester (make sure to checkout feature/authentication branch)
-  Best to just set environment variable manually to true 
```
    ....
    harvester: {
      authEnabled: true,
    },
```
- see if you get login screen
- login and see if you can make authenticated jobs/scheduled jobs
